src/main/java/com/atlassian/jira/cloud/jenkins/buildinfo/client/BuildsApi.java

```diff
@@ -0,0 +1,41 @@
+package com.atlassian.jira.cloud.jenkins.buildinfo.client;
+
+import com.atlassian.jira.cloud.jenkins.buildinfo.client.model.BuildApiResponse;
+import com.atlassian.jira.cloud.jenkins.buildinfo.client.model.Builds;
+import com.atlassian.jira.cloud.jenkins.common.client.ApiUpdateFailedException;
+import com.atlassian.jira.cloud.jenkins.common.client.JenkinsAppApi;
+import com.atlassian.jira.cloud.jenkins.common.client.JenkinsAppRequest;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.OkHttpClient;
+
+public class BuildsApi extends JenkinsAppApi<BuildApiResponse> {
```

Comment:
Previously, there was a single `JiraApi`. I split this out into dedicated `BuildsApi` and `DeploymentsApi` to have less generic code.